src/Framework/MockObject/Generator/Generator.php

```diff
@@ -763,7 +763,7 @@ private function generateCodeForTestDoubleClass(string $type, bool $mockObject,
         }
 
         if ($mockMethods->hasMethod('method') || (isset($class) && $class->hasMethod('method'))) {
-            $message = 'Doubling interfaces (or classes) that have a method named "method" is deprecated. Support for this will be removed in PHPUnit 12.';
+            $message = 'Doubling interfaces (or classes) that have a method named "method" is deprecated and will be removed in PHPUnit 12 without replacement.';
```

Comment:
👎 I am sorry, but I think the current message is better.

Reply:
Ack, removed from PR